tests/Common/BranchEventsTest.php

```diff
@@ -207,7 +209,7 @@ public function testFilterBranchTokenEventsIsNotInDefaultBranch(): void
 
         // test DEFAULT branch
         // check token events in default branch
-        $defaultTokenEvents = $this->_client->listTokenEvents($token['id']);
+        $defaultTokenEvents = $this->_client->listTokenEvents($token['id'], ['sinceId' => $lastEventId]);
```

Comment:
Tady na tom to spadlo jakože nějakou race condition, nebo proč jsi to přidal?

Reply:
Neprochazelo mi to lokalne, protoze to vracelo i starsi. Uz jsem v minulosti proto nekolik testu upravoval.